players.js

```diff
@@ -1,6 +1,7 @@
-const {Clutter,Gio,GLib,GObject,Shell,St} = imports.gi;
+const {Clutter,Gio,GLib,GObject,Shell,St,Meta} = imports.gi;
```

Comment:
is `Meta` used?

Reply:
Not anymore, I tested a method which I thought might might need it to define normal windows but that didn't work. It can be removed.